openwisp-qa-check

```diff
@@ -188,6 +200,12 @@ while [ "$1" != "" ]; do
   --skip-checkcommit)
     SKIP_CHECKCOMMIT=true
     ;;
+  --skip-stylelint)
+    SKIP_STYLELINT=true
+    ;;
+  --skip-jshint)
+    SKIP_JSHINT=true
```

Comment:
Great to see that we are adding this in this script! Will be great for reusability.

But we definitely should not do it this way @niteshsinha17 @atb00ker, because it will cause some existing builds to fail because these linters may not be installed.

We must introduce this feature in a backward compatible way, so either by default these linters should not run (the options could be something like ``--css`` and ``--js`` to activate the respective linters) or before running the shell script should verify the availability of the linter program and skip if it's not present.

Moreover, we'll have to update the ``show_help`` function above in this file and the README of the module to mention these options.

Reply:
You are right. I was also thinking about this.